math/acl2/Portfile

```diff
@@ -32,10 +32,11 @@ long_description    \
     as the language (a subset of Common Lisp), it is very \
     flexible.
 
-# user_notes Users who want to use ACL2 for serious work should
-# install the certify variant (sudo port install +certify),
-# which will certify (i.e., prove all of the theorems)
-# in the included examples. This can take several hours.
+notes "Users who want to use ACL2 for serious work should
```

Comment:
The other uses of `notes` in this Portfile need to be changed to `notes-append`, or they will overwrite this.

Reply:
Thanks for pointing this out :) Fixed it now. Will keep these things in mind in future.